services/locationClient.js

```diff
@@ -29,24 +29,31 @@ import {startRangingBeacons} from '../services/beaconManagerClient';
       startRangingBeacons(saveDeviceProximity);
     };
 
-    const getLocation = () => {
-      //TODO saveLocation should be inside getLocation in backend.
-      saveLocation();
-
-      //TODO call service api call when available.
-      //startRangingBeacons(getLocationApiCall);
-
-      //TODO this is fakeLocation Replace it with service when available.
-      return  [{
-        id: "ldksfjdslkf",
-        center: {
-          latitude: -34.6403200,
-          longitude: -58.401555,
+    const getLocationApiCall = (beacons) => {
+      return fetch(GET_LOCATION_SERVICE_URL, {
+        method: 'POST',
+        headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
         },
-        radius: 1,
-      }];
+        body: JSON.stringify({
+          device_id: getUniqueId(),
+          nearby_stands: getNearbyStands(beacons)
+        }),
+      })
+      .then((response) => response.json())
+      .then((responseJson) => {
+        return responseJson
+      })
+      .catch((error) => {
+        console.error(error);
```

Comment:
Evitemos romper en development por alguna demo
```suggestion
        console.log(error);
```